src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/Compilation.cs

```diff
@@ -691,6 +691,8 @@ public IEnumerable<TypeDesc> ConstructedEETypes
                 }
             }
         }
+
+        public DependencyAnalyzerBase<NodeFactory> Graph => _graph;
```

Comment:
Could you expose this on `CompilationResults` instead? If the whole graph is public, there's no point to have `CompilationResults`.

I kind of like that IEETypeNode and IMethodBodyNode are implementation details. Going over the graph exposes e.g. the duality of Constructed/Unconstructed types (some types will appear twice).

Reply:
(I mean expose a way to get to the list of all method tables constructed or not. I don't mean a way to get to every marked node.)